homeassistant/components/glances/const.py

```diff
@@ -13,7 +13,4 @@
 DATA_UPDATED = "glances_data_updated"
 SUPPORTED_VERSIONS = [2, 3]
 
-if sys.maxsize > 2**32:
-    CPU_ICON = "mdi:cpu-64-bit"
-else:
-    CPU_ICON = "mdi:cpu-32-bit"
+CPU_ICON = f"mdi:cpu-{64 if sys.maxsize > 2**32 else 32}-bit"
```

Comment:
This change is to get 100% coverage in `const.py`, and therefore remove it from `.coveragerc`